### ISSUE TITLE
Some accounts can not be logged in due to network issues

### ISSUE THREAD

**Issue by Tracachang** [Blockstream/green_qt#140]
### Description

Liquid accounts are not displayed when using tor, this has been happening for a few days, also on desktop and android.  A red message appears saying "Some accounts may not be able to log in due to network issues.Please try again later". Partial service outage.

### Version

2.0.9

### Steps to reproduce

in "settings", I enable "connect with tor" and in "custom and servers and validation", I enter my "bitcoin electrum server" and leave the "liquid electrum server" empty.

### Expected behaviour

I should be able to interact with liquid account as before.

### Actual behaviour

In the desktop version, liquid accounts are not displayed and appears  the error message "Some accounts may not be able to log in due to network issues.Please try again later", in the Android version they are displayed but not updated.

### Device or machine

ManjaroLinux 24.0.7

**Comment by borgbyte** [comment]
Doesn't work if you click "try again"?

**Comment by Tracachang** [comment]
> Doesn't work if you click "try again"? Does it work with Tor disabled?

No, it does not work when I click "try again", and yes, it works with Tor disabled.

**Comment by Tracachang** [comment]
Everything is back to normal and working as expected.

**Comment by Tracachang** [comment]
> I too have this persistent issue when TOR is connected. MacOS running 2.0.15

I also have this problem again.

**Comment by 91-xTx-93** [comment]
I am experiencing the same connection issues with IVPN, but there are no problems when I am not using it.

**Comment by emmanuelrosa** [comment]
I had this problem, but release 2.0.21 fixed it.

**Comment by hgs-sellis** [comment]
I have this problem consistently on 2.0.25
